packages/typescript-estree/src/node-utils.ts

```diff
@@ -611,21 +635,39 @@ export class TSError extends Error {
       configurable: true,
     });
   }
+
+  get index(): number {
+    return this.location.start.offset;
+  }
+
+  get lineNumber(): number {
+    return this.location.start.line;
+  }
+
+  get column(): number {
+    return this.location.start.column;
+  }
```

Comment:
Should those kept?

Reply:
Is this the answer? https://github.com/eslint/eslint/blob/b09a512107249a4eb19ef5a37b0bd672266eafdb/lib/linter/linter.js#LL853C32-L853C32

Reply:
that's correct, we have to keep `lineNumber` and `column` as they are used in eslint
as for `index` if i remember correctly, this prop was needed in older version of eslint